lib/modules/showImages.js

```diff
@@ -1902,6 +1994,38 @@ addModule('showImages', (module, moduleID) => {
 		}
 	};
 
+	module.rotateMedia = function(ele, rotationState) {
```

Comment:
If this isn't used outside the module, I'd prefer to have a bare `function rotateMedia(ele, rotationState) {`.
We can add `module.rotateMedia = rotateMedia;` in the future if we need to expose it (or hopefully I'll be done converting to ES6 modules by then).